block-layer/codecs/dag-cbor.md

```diff
@@ -0,0 +1,41 @@
+# Specification: DagCBOR
+
+**Status: Descriptive - Draft**
+
+DagCBOR supports the full [IPLD Data Model].
+
+CBOR already natively supports all [IPLD Data Model Kinds].
+
+## Format
+
+The CBOR IPLD format is called DagCBOR to disambiguate it from regular CBOR.
+Most CBOR objects are valid DagCBOR. The only hard restriction is that any field
+with the tag 42 must be a valid CID.
+
+## Link Format
+
+As with all IPLD formats, DagCBOR must be able to encode [Links].
+In DagCBOR, links are encoded using the raw-binary identity [Multibase] in a
```

Comment:
original:
>  DagCBOR, links are encoded using the raw-binary (identity, NUL) multibase in a field with ...

The `(identity, NUL)` bit is probably important to keep, or at least highlight in some way, as @warpfork found out [recently](https://github.com/ipld/go-ipld-prime/pull/23), although it's worded very unclearly. Adding a bracketed clarification might be enough, something like "(effectively prepending a `0` byte to the CID bytes)".

Reply:
My idea was that we are referring to "identity" in the multibase spec, so it should be clear. But I'm happy adding it back in.

Reply:
@rvagg Would this be clearer:

In DagCBOR, links are [CIDs] encoded using the raw-binary identity [Multibase], i.e. the raw bytes prefixed with `0x00`. They are stored as byte-string type (major type 2), with the tag 42.